cmd/bootstrap/cmd/key_test.go

```diff
@@ -15,6 +15,17 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+func TestGenerateKeys(t *testing.T) {
```

Comment:
Looking in this file it actually looks like there is already [some tests set up](https://github.com/onflow/flow-go/pull/722/files#diff-fd992efa53b7e741bbf99ff01fce62a02af4798fc9c5dd075489d461efc6e7dcR39) which hook right into the cobra command and test a larger surface area than what I suggested earlier. We can add an assertion there that the machine account key file is created. 

Probably best to follow the lead of these existing tests for the subsequent commands as well.